### ISSUE TITLE
Some types are not converted

### ISSUE THREAD

**Issue by pathmann** [pathmann/PythonQt#2]
For some types (eg QModelIndexList), PythonQt will fail to convert (PythonQtConvertListOfKnownClassToPythonList) to a pythonlist (getClassInfo == NULL), and crash the hosting application.

How to reproduce:
```python3
from PythonQt.QtGui import *
t = QTableWidget()
s = t.selectionModel()
print(s.selectedIndexes()) #this will fail 
```

**Comment by pathmann** [comment]
Better testcase:
```python
from PythonQt.QtGui import *
t = QTableView()
m = QStandardItemModel()
t.setModel(m)
s = t.selectionModel()
print(s.selectedIndexes())
```

**Comment by pathmann** [comment]
Coming closer. Unlike eg QWidgetList (=> QList<QWidget*>), QModelIndexList is not converted to QList<QModelIndex>, so PythonQtConvertListOfKnownClassToPythonList will fail to find the right listtype (PythonQtMethodInfo::getInnerTemplateTypeName won't return QModelIndex, but an empty ByteArray instead).

**Comment by pathmann** [comment]
Fixed with 1a884de.
